docs/docker.md

```diff
@@ -40,8 +40,8 @@ To run management commands, create a temporary container with the same
 /data directory and put the command after the image name, like this:
 
 ```
-docker run --rm -it -v maddydata:/data foxcpp/maddy:0.6.0 creds create foxcpp@maddy.test
-docker run --rm -it -v maddydata:/data foxcpp/maddy:0.6.0 imap-acct create foxcpp@maddy.test
+docker run --rm -it -v maddydata:/data foxcpp/maddy:latest creds create foxcpp@maddy.test
```

Comment:
I don't want to encourage use of `latest` tag since this means people can inadvertently install a version with backward-incompatible changes. At best, `0.Y` tags (0.7) should be used.